jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+
+- link:https://github.com/users/timja/projects/4[Core maintainers attention]
+- link:https://github.com/users/timja/projects/3[User experience project]
+
+Issues need to be added to a project this can be achieved with a script link:add-issue-to-project.sh[].
+
+The dashboards aren't as powerful as Jira and the sorting is limited. See link:https://github.com/github-community/community/discussions/8518[Can't currently sort issues by created / updated].
+
+==== Subscribing to select issues, e.g. label
+
+Users in Jira were able to configure notifications based off of a saved query, which was a very powerful feature.
+GitHub's built-in feature only allows you to subscribe to all issues or subscribe individually as you see an issue.
+
+There are a couple of workarounds:
+- link:https://github.com/orgs/github-community/discussions/16645[Subscribe to labels in GitHub]
+- link:https://github.com/bytecodealliance/subscribe-to-label-action[]
+- link:https://github.com/rseanhall/issue-label-watcher[]
+
+==== Project management in one view
+
+Previously Jira was used to track any bigger projects such as Guava upgrade and Java 11.
+This was complicated by some plugins using GitHub and some Jira to track issues.
+
+As of June 2022 515 repositories have GitHub issues, (21% of repositories). 46325 issues have been file in GitHub.
+
+GitHub projects can be used to group issues that are across components.
+If that project doesn't use GitHub issues then either:
+- A tracking issue can be created that links to a Jira epic or query
+- A repo is created for tracking individual issues and those issues link to an issue in the plugins issue tracker
+
+=== Missing matching capability from Jira
+
+==== Can't move issues without write access on both sides
+
+In Jira to move an issue to a more appropriate component then the component field was updated.
+
+In GitHub to move an issue you can either:
+1. Transfer it if you have write access on both repositories
+2. Refile the issue under a different user linking back to the old one.
+
+The Jira workflow relied on:
+1. Users updating the assignee so that default assignee logic was re-evaluated and the actual maintainer notified
+2. Maintainers manually configuring filters in Jira to notify them
+
+If we do wish to maintain the ability to move issues for anyone or at least org members then we would need to create a small app that has the ability to do this, deploy it somewhere, and configure a webhook at the organization level.
+
+A simpler approach could be a GitHub action in the jenkinsci/core repository but this would mean it would only work in that repository and not others.
+
+The Kubernetes link:https://www.kubernetes.dev/docs/guide/issue-triage/#abandoned-or-wrongly-placed-issues[closes misfiled issues].
+
+==== Cannot mark one GitHub issue as caused by or depending on another GitHub issue
+
+Jira allows explicit relationships to be set on links.
+GitHub requires you to type 'Caused by' and 'Depends on' yourself.
```

Comment:
There is a GHA for this: https://github.com/marketplace/actions/dependent-issues

Reply:
oh cool, could be useful